components/Logo/Logo.js

```diff
@@ -0,0 +1,41 @@
+import styled from "styled-components";
+
+function Logo() {
+  return (
+    <LogoContainer>
+      <LogoOrange>P</LogoOrange>
+      <LogoPink>a</LogoPink>
+      <LogoGreen>n</LogoGreen>
+      <LogoOrange>t</LogoOrange>
+      <LogoGreen>r</LogoGreen>
+      <LogoPink>y</LogoPink>
+    </LogoContainer>
+  );
+}
+
+export default Logo;
+
+const LogoContainer = styled.div`
+  display: flex;
+  justify-content: center;
+`;
+
+const LogoOrange = styled.h2`
+  font-family: "Kalam", cursive;
+  font-size: 2.5rem;
+  margin: 1.5rem 0;
+  color: var(--orange);
+`;
+
+const LogoPink = styled.h2`
+  font-family: "Kalam", cursive;
+  font-size: 2.5rem;
+  margin: 1.5rem 0;
+  color: var(--pink);
+`;
+const LogoGreen = styled.h2`
+  font-family: "Kalam", cursive;
+  font-size: 2.5rem;
+  margin: 1.5rem 0;
+  color: var(--lightgreen);
+`;
```

Comment:
For semantics, i'd suggest using one `h2` as the container and `span` elements for coloring the letters.
Common properties like `font-family`, `font-size` and `margin` can be moved to the container:

```suggestion
const LogoContainer = styled.h2`
  display: flex;
  justify-content: center;
  font-family: "Kalam", cursive;
  font-size: 2.5rem;
  margin: 1.5rem 0;
`;

const LogoOrange = styled.span`
  color: var(--orange);
`;

const LogoPink = styled.span`
  color: var(--pink);
`;

const LogoGreen = styled.span`
  color: var(--lightgreen);
`;
```